src/library/baseexternalplaylistmodel.h

```diff
@@ -1,16 +1,18 @@
 #pragma once
 
-#include <QtSql>
 #include <QItemDelegate>
-#include <QString>
-#include <QObject>
 #include <QModelIndex>
+#include <QObject>
+#include <QString>
+#include <QStringBuilder>
```

Comment:
not necessary according to https://doc.qt.io/qt-5/qstring.html#more-efficient-string-construction
`QT_USE_QSTRINGBUILDER` is set in https://github.com/mixxxdj/mixxx/blob/8715006244ffcafbc6e3ef365ec832a114104626/CMakeLists.txt#L2116